workspace_tools/tests.py

```diff
@@ -557,6 +557,14 @@
         "automated": True,
         #"host_test": "wait_us_auto"
     },
+    {
+        "id": "MBED_35", "description": "SPI C12832 display",
+        "source_dir": join(TEST_DIR, "mbed", "spi_C12832"),
+        "dependencies": [MBED_LIBRARIES, TEST_MBED_LIB, join(PERIPHERALS, 'C12832')],
```

Comment:
My bad!
You are right, I forgot about peripheral libs. You've followed correct way to do it :)